configuration.json

```diff
@@ -0,0 +1,23 @@
+{
+ "input_matrix_path": "/Users/flaminia/Documents/work/futureforest_landshut/input_matrices_exp1/input_matrix_ru_id_1.csv",
```

Comment:
@mspada After a discussion with Kapil, we agreed to leave the relative paths as they are for now. The reason is that either the user gives the relative paths in the configuration file (like now), or she/he has to copy the input data somewhere specified in the readme file (like it would be if we use absolute paths).